src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -3510,7 +3513,7 @@ export class StudyViewPageStore
                 const uniqueKey = item.attributeId;
                 if (this.isNewlyAdded(uniqueKey)) {
                     this.showAsPieChart(uniqueKey, item.counts.length);
-                    this.newlyAddedCharts.remove(uniqueKey);
+                    //this.newlyAddedCharts.remove(uniqueKey);
```

Comment:
commenting out these removals (there are 4) seems to fix the problem, except of course that newlyAddedCharts doesn't get cleared.